Index: src/main/java/com/example/SenateAnalysis.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/SenateAnalysis.java b/src/main/java/com/example/SenateAnalysis.java
new file mode 100644
--- /dev/null	(date 1612814251544)
+++ b/src/main/java/com/example/SenateAnalysis.java	(date 1612814251544)
@@ -0,0 +1,4 @@
+package com.example;
+
+public class SenateAnalysis {
+}
Index: src/main/java/com/example/SenateFiltering.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/SenateFiltering.java b/src/main/java/com/example/SenateFiltering.java
new file mode 100644
--- /dev/null	(date 1612814251527)
+++ b/src/main/java/com/example/SenateFiltering.java	(date 1612814251527)
@@ -0,0 +1,4 @@
+package com.example;
+
+public class SenateFiltering {
+}
